src/consensus/pbft/pbft.h

```diff
@@ -106,19 +108,22 @@ namespace pbft
     enclave::RPCSessions& rpcsessions;
     std::unique_ptr<consensus::LedgerEnclave> ledger;
     SeqNo global_commit_seqno;
+    std::unique_ptr<pbft::Store> store;
 
   public:
     Pbft(
       std::shared_ptr<ChannelProxy> channels_,
       NodeId id,
       std::unique_ptr<consensus::LedgerEnclave> ledger_,
       std::shared_ptr<enclave::RpcMap> rpc_map,
-      enclave::RPCSessions& rpcsessions_) :
+      enclave::RPCSessions& rpcsessions_,
+      std::unique_ptr<pbft::Store> store_) :
       Consensus(id),
       channels(channels_),
       rpcsessions(rpcsessions_),
       ledger(std::move(ledger_)),
-      global_commit_seqno(0)
+      global_commit_seqno(0),
```

Comment:
`global_commit_seqno` is currently incremented for every transaction locally committed (line 311). Instead, just like for Raft, we should update this variable when the store is compacted, i.e. in the new `global_commit_cb`. 

Also, could we rename `global_commit_seqno` to `commit_idx` to be consistent with Raft?

Reply:
I don't think the terminology needs to be consistent with Raft here?

Reply:
You're right. I forgot we already had a nice adaptor class to keep the consensus interface generic: https://github.com/microsoft/CCF/blob/master/src/consensus/raft/raftconsensus.h#L69